### ISSUE TITLE
Who is Tim Matsumoto... and we should probably update the README

### ISSUE THREAD

**Issue by davibarreira** [JuliaOptimalTransport/OptimalTransport.jl#94]
This issue is to deal with the README. I see that our documentation is already very far ahead from the README. So we should probably update it. I've been thinking about doing it, but I was wondering what you guys think it should have (usage examples with figures? list of functionalities? a bit of Optimal Transport theory?).

P.S: Who is Tim Matsumoto?

**Comment by devmotion** [comment]
I would suggest reducing redundant information and pointing users mainly to the documentation (which could be improved and extended, of course). Probably it is still useful to quickly summarize the package in the README (without explicitly listing individual functions) and to keep the list of related packages and instructions for new contributors. A very short example of code (without (almost) any explanations) and/or a plot could be nice but I think anything more detailed should be added to the documentation instead.

**Comment by davibarreira** [comment]
Cool. So we should then get rid of that "Overview of supported functionality" and replace by the quick summary and the example case. I believe your suggestion would be something similar to your repo [here](https://github.com/devmotion/PalmerPenguins.jl).

**Comment by devmotion** [comment]
No, that's an example of a package where the README is the only and full documentation. I had something in mind like https://github.com/TuringLang/MCMCChains.jl, https://github.com/JuliaGaussianProcesses/KernelFunctions.jl, or https://github.com/JuliaGaussianProcesses/AbstractGPs.jl

**Comment by zsteve** [comment]
Tim Matsumoto implemented the current quadratic regularisation solver (which also needs to be refactored)

I agree about really needing to update the README. I agree with @devmotion, we should have probably a brief overview of the supported functionality and a short 'quick start' guide for the most basic tasks. Don't need to explain the theory (this might be good for the docs though!)

**Comment by davibarreira** [comment]
Thanks @zsteve. I was wondering who he was since he does not appear as a github contributor to the package.
